geos/entrypoints.go

```diff
@@ -286,3 +286,15 @@ func CoverageUnion(g geom.Geometry) (geom.Geometry, error) {
 func UnaryUnion(g geom.Geometry) (geom.Geometry, error) {
 	return rawgeos.UnaryUnion(g)
 }
+
+// ConcaveHull returns concave hull of input geometry.
+// pctconvex - ratio 0 to 1 (0 - max concaveness, 1 - convex hull)
+// allowHoles - true to allow holes inside of polygons.
+func ConcaveHull(g geom.Geometry, pctconvex float64, allowHoles bool) (geom.Geometry, error) {
```

Comment:
Nit: could you rename `pctconvex` to either `ratio` or maybe `convexityRatio`? The `pct` prefix isn't really right because it's a ratio (0 to 1) rather than a percentage (0 to 100). As a reference, the GEOS interface uses `ratio` as the name.

Reply:
I found ratio is confusing here, param named similar to https://postgis.net/docs/ST_ConcaveHull.html

Reply:
addressed review comments